content/services/development.md

```diff
@@ -123,9 +147,9 @@ that express the values of your company.
 
 ### Languages & Tools
 
-- Python **with** Django (*core contributor*)
-- HTML/CSS **with** Sass (*core contributor*)
-- JavaScript **with** Vue & React
+- **Python** with **Django** (*core contributor*)
+- **HTML/CSS** with **Sass** (*core contributor*)
+- **JavaScript** with **Vue** & **React**
```

Comment:
@jgerigmeyer should we add Svelte to this list?
```suggestion
- **JavaScript** with **Vue**, **React**, or **Svelte**
```

Reply:
I'm fine with that, even if it's more aspirational than anything. 😄